src/rsz/src/RepairHold.cc

```diff
@@ -440,7 +442,16 @@ void RepairHold::repairEndHold(Vertex* end_vertex,
               }
             }
           }
-          if (!load_pins.empty()) {
+          // check for dont touch loads
```

Comment:
Rather than prevent buffering if there is a dont-touch fanout you can just leave the fanouts attached and add the new net to the buffer's input.  I believe RepairDesign::makeRepeater does this and perhaps could be generalized?